packages/panels/resources/views/components/sidebar/index.blade.php

```diff
@@ -83,7 +83,12 @@ class="-mx-1.5 ms-auto hidden lg:flex"
         @endif
     </header>
 
-    <nav class="fi-sidebar-nav flex flex-col gap-y-7 overflow-y-auto px-6 py-8">
+    <nav 
+        class="fi-sidebar-nav flex flex-col gap-y-7 overflow-y-auto overflow-x-hidden px-6 py-8"
+        @if (filament()->isSidebarCollapsibleOnDesktop())
+            x-bind:class="$store.sidebar.isOpen ? '' : 'items-center'"
```

Comment:
Could you refactor this to `{ 'items-center': ... }`?

Reply:
Good call. Done